This test relies on 'xargs' being available in $PATH, which is not
the case when we build the initial Findutils doing bootstrapping.
Reported at <https://savannah.gnu.org/bugs/index.php?46786>.

--- findutils-4.6.0/find/testsuite/sv-34976-execdir-fd-leak.sh	2015-12-31 19:37:59.401526288 +0100
+++ findutils-4.6.0/find/testsuite/sv-34976-execdir-fd-leak.sh	2015-12-31 19:38:36.061770693 +0100
@@ -50,13 +50,14 @@ die() {
 # Create test files, each 98 in the directories ".", "one" and "two".
 make_test_data() {
   d="$1"
+  xargs="`cd ../../xargs; pwd -P`/xargs"
   (
     cd "$1" || exit 1
     mkdir one two || exit 1
     for i in ${three_to_hundred} ; do
       printf "./%03d one/%03d two/%03d " $i $i $i
     done \
-      | xargs touch || exit 1
+      | "$xargs" touch || exit 1
   ) \
   || die "failed to set up the test in ${outdir}"
 }
